source/parser.hera

```diff
@@ -5677,6 +5686,16 @@ Init
       }
     }
 
+    function processTryExpressions(statements) {
+      if (module.config.implicitReturns) {
+        // Add returns to TryExpressions
+        gatherRecursiveAll(statements, n => n.type === "TryExpression")
+        .forEach(({ block }) => {
+          insertReturn(block)
```

Comment:
The catch block also needs returns inserted (if present).

Reply:
This should be done at this point.